extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -36,11 +42,29 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 			},
 			slotName = 'INCONTENT_BOXAD_1',
 			startPosition = placeHolderElement.offsetTop,
-			swapRecirculationAndAd;
+			swapRecirculationAndAd,
+			btRec = null,
+			recNode = null;
 
 		adSlot.className = 'wikia-ad';
 		adSlot.setAttribute('id', slotName);
 
+		function applyRec() {
+			if (btRec === null) {
+				btRec = babDetection.isBlocking() && wadRecRunner.isEnabled('bt');
+			}
+			recNode = btRecLoader.duplicateSlot(slotName);
```

Comment:
Let's put it inside `if (btRec) {}` because we don't wanna duplicate the slot is there is no adblock :)